editor.planx.uk/src/theme.ts

```diff
@@ -38,6 +41,15 @@ const DEFAULT_PALETTE: Partial<PaletteOptions> = {
     primary: "#0B0C0C",
     secondary: "#505A5F",
   },
+  link: {
+    main: DEFAULT_PRIMARY_COLOR
+  },
+  prompt: {
+    main: DEFAULT_PRIMARY_COLOR,
+    contrastText: "#FFFFFF",
+    light: lighten(DEFAULT_PRIMARY_COLOR, DEFAULT_TONAL_OFFSET),
+    dark: darken(DEFAULT_PRIMARY_COLOR, DEFAULT_TONAL_OFFSET),
+  },
```

Comment:
There is an `augmentColor()` utility to handle this automatically, but it only happens _after_ a theme has been generated so I'm doing this manually here.

Docs: https://mui.com/material-ui/customization/palette/#generate-tokens-using-augmentcolor-utility